ports/esp32/boards/LILYGO_TTGO_LORA32/modules/lora32.py

```diff
@@ -0,0 +1,79 @@
+"""LILYGO TTGO LoRa32 MicroPython Helper Library."""
+
+from machine import Pin, SoftI2C, Signal
+
+from lilygo_oled import OLED
+
+from micropython import const
+
+
+class Lora32Base:
+    """Base class defining common pins."""
+
+    def __init__(self, define_helpers=True):
+        # LORA
+        self.LORA_MOSI = const(27)
+        self.LORA_MISO = const(19)
+        self.LORA_SCLK = const(5)
+        self.LORA_CS = const(18)
+        self.LORA_DIO = const(26)
+        self.LORA_RST = const(23)
+
+        # DAC
+        self.DAC1 = const(26)
+
+        # LED
+        self.LED = const(25)
+
+        # OLED
+        self.OLED_SDA = const(21)
+        self.OLED_SCL = const(22)
+
+        if define_helpers:
+            self.create_helpers()
+
+    def create_helpers(self):
+        self.led = Signal(self.LED, Pin.OUT, value=0)
```

Comment:
I think you can just use `Pin` here, instead of `Signal`.

Reply:
Done

Originally did this as I thought the interface led.on() and led.off() was more intuitive, but if it's not standard for these included modules then no worries, have used Pin.

Reply:
Pin also has on/off methods.

Reply:
Thanks, missed that invert is the difference, not the on/off method.